### PR TITLE
Handle empty string fields from POM

### PR DESCRIPTION
**Pull request by eygraber**
- See [tweetnacl-java](https://repo.maven.apache.org/maven2/org/purejava/tweetnacl-java/1.1.2/tweetnacl-java-1.1.2.pom) as an example

### CODE REVIEW

**Review by loosebazooka — APPROVED** [review]
lgtm, just a small naming thing

**Comment by eygraber** [comment]
Did you leave a comment about the naming thing? I don't see any

File: src/main/java/org/spdx/sbom/gradle/maven/PomResolver.java [hunk 1]
```diff
@@ -173,16 +173,27 @@ public Map<String, PomInfo> effectivePoms(List<ResolvedArtifactResult> resolvedP
...
+                                  .email(presentIfNotEmpty(d.getEmail()))
+                                  .organization(presentIfNotEmpty(d.getOrganization()))
                                   .build())
                       .collect(Collectors.toList()))
               .build());
     }
     return effectivePoms;
   }
 
+  private Optional<String> presentIfNotEmpty(String s) {
```
Comment (loosebazooka) [inline src/main/java/org/spdx/sbom/gradle/maven/PomResolver.java:186]: this is personal preference: presentIfNotEmpty -> OptionalOfNonEmpty

Comment (eygraber) [reply src/main/java/org/spdx/sbom/gradle/maven/PomResolver.java]: Updated

**Pull request merged**